src/nanoemoji/config.py

```diff
@@ -168,18 +169,39 @@ def output_format(self):
         return Path(self.output_file).suffix
 
     @property
-    def has_bitmaps(self):
-        return self.color_format.startswith("sbix") or self.color_format.startswith(
-            "cbdt"
+    def has_bitmaps(self) -> bool:
+        return bool({"sbix", "cbdt"}.intersection(self.color_format.split("_")))
+
+    @property
+    def has_picosvgs(self) -> bool:
+        return bool(
+            {"glyf", "colr", "picosvg", "picosvgz"}.intersection(
+                self.color_format.split("_")
+            )
+        )
+
+    @property
+    def has_untouchedsvgs(self) -> bool:
+        return bool(
+            {"untouchedsvg", "untouchedsvgz"}.intersection(self.color_format.split("_"))
         )
```

Comment:
Looking at how these methods are evolving makes me think the color format should become a class with properties rathe rthan a string. Out of scope here I suppose.

I think an internal _has would help here, then you get

```python
    def has_bitmaps(self) -> bool:
        return _has_any("sbix", "cbdt")
```

Reply:
I'll keep as a string for now, but I like the idea of adding the `_has_any` helper